src/main/java/com/exasol/adapter/adapternotes/ColumnAdapterNotes.java

```diff
@@ -68,14 +70,14 @@ public static ColumnAdapterNotes deserialize(final String columnAdapterNotes, fi
             throws AdapterException {
```

Comment:
This class needs refactoring. The column name is passed just to have something for the exception messages. Seems like unnecessary detail disclosure at this point.

But: not now. Please write an issue ticket.

Reply:
created ticket